packages/@glimmer/addon-blueprint/files/ember-addon-index.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const Funnel = require('broccoli-funnel');
+const BroccoliDebug = require('broccoli-debug');
+const MergeTrees = require('broccoli-merge-trees');
+
+module.exports = {
+  name: '<%= name %>',
+  blueprintsPath() {
+    return null;
+  },
+  preprocessTree(type, tree) {
```

Comment:
I'm not sure if we should make this in the default blueprint. This style worked well for your polyfill addon because we ultimately expect those helpers to be moved into "core", but in general with MU we don't want folks mutating the source apps `src` 🤔

I think the better thing to do (until we have MU support for `{{use` or the like) is generate the files into the host. 

What do you think?

Reply:
I'm not sure.  I think the current `use` RFC isn't in a place where we can really use it.  And there are still questions about it.  

I think, so long as we have an ability to automatically move people in a different direction, we'd prefer this approach to polutting consuming applications with a million files.

Maybe making `use` work in block form (which as we discussed might be possible in user space) would be a temporary hack that'd still give us namespace.

Reply:
I definitely understand your point, but I am still uncomfortable shipping an official addon blueprint that mutates the host app's `src`. 🤔